net/siproxd/src/opnsense/mvc/app/models/OPNsense/Siproxd/User.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace OPNsense\Siproxd;
+
+use OPNsense\Base\BaseModel;
```

Comment:
I can remember that @fichtner asked me to move that after the license header some time ago.

Reply:
fixed